### ISSUE TITLE
Digital Ocean test script which runs in parallel for JSX 10.0.1

### ISSUE THREAD

**Issue by despiegk** [threefoldtech/jumpscaleX#768]
- start from existing config manager DO client
- deploy machine builder

...

see https://docs.grid.tf/threefold/itenv_web_v2/issues/2

**Comment by ranatrk** [comment]
current status in issue :https://docs.grid.tf/threefold/itenv_web_v2/issues/2 
waiting for feedback from @despiegk  to resume working on issue and testing on digital oceam.

**Comment by despiegk** [comment]
ee also #596

**Comment by rkhamis** [comment]
no longer relevant
